misc/mzutil/snapshot_view_states.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env python3
+# Copyright Materialize, Inc. All rights reserved.
+#
+# Use of this software is governed by the Business Source License
+# included in the LICENSE file at the root of this repository.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0.
+
+"""snapshot_view_states
+
+Script to query all materialized views in a Materialize database and persist a point in time copy
+of each view to files in the snapshots directory.
+
+Best used when no data is running through the system.
+"""
+
+import argparse
+import os
+
+import psycopg2
+
+
+def view_names(conn):
```

Comment:
Can this (and the identical method in the other file) be extracted into a common file?